### PR TITLE
fix: use supported-versions variables in ceph-api

### PR DESCRIPTION
**Pull request by mdelapenya**
## What does this PR do?
It uses the supported-versions env variables for the ceph-api service in the docker-compose.yml file for the integration.

## Why is it important?
It seems values are hardcoded, and that could be affecting the E2E tests.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Related issues

- _Probably_ caused by https://github.com/elastic/beats/commit/273c6a896a6497986842e23bb571d9f39a95fac4
- Relates elastic/e2e-testing#618
- Discovered while testing elastic/e2e-testing#630

### CODE REVIEW

File: metricbeat/module/ceph/docker-compose.yml [hunk 1]
```diff
@@ -13,11 +13,11 @@ services:
       - 8003
       - 8080
   ceph-api:
-    image: docker.elastic.co/integrations-ci/beats-ceph:master-6373c6a-jewel-centos-7-x86_64-1
+    image: docker.elastic.co/integrations-ci/beats-ceph:${CEPH_VERSION:-master-97985eb-nautilus-centos-7-x86_64}-1
```
Comment (mdelapenya) [inline metricbeat/module/ceph/docker-compose.yml:16]: I'm not sure why this image is version `-1` and the regular ceph is `-2`. @jsoriano could you please 👀 here?

**Review by cachedout — APPROVED** [review]

File: metricbeat/module/ceph/docker-compose.yml (hunk 1 above)
Comment (jsoriano) [inline metricbeat/module/ceph/docker-compose.yml:16]: I don't think we can use the same versions in both cases, if I remember well, we kept this here to keep some tests specifically running against some specific older version.

I was wondering how is it possible then that tests pass after the changes in this PR, and I have found that there isn't actually anything using at the moment with the `ceph-api` service. It is not used in the python system tests, and go integration tests are skipped as flaky: https://github.com/elastic/beats/issues/22612

Changes here probably need to be deeper, and maybe not only on the testing side. I think that we need to fix the integration tests and check what system tests are actually testing. Maybe at the end we can use a single `ceph` service, but we need to make E2E and go integration tests smarter so they know what metricsets they should be using for each versions. This is what the system tests seem to be doing.

Or maybe we can refactor the metricsets so they detect the version and do nothing if they implement collection methods that are not supported. This way we could enable all metricsets with any version. But I'd say that changes in this module have low priority at the moment.

Comment (jsoriano) [reply metricbeat/module/ceph/docker-compose.yml:16]: I think we wanted to keep using an specific version here, that is different to the one in the `ceph` service.

**Review by jsoriano — CHANGES_REQUESTED** [review]

**Comment by jlind23** [comment]
PRs grooming - Closing for now until further updates.

**Pull request closed without merging**